tools/github_actions_source_hash_updater.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
If we're going to be splitting everything out into smaller scripts, would it make sense to have a tools/actions directory? Right now the tools directory is getting pretty cluttered with scripts that are really only used in CI.